internal/app/app.go

```diff
@@ -230,6 +230,10 @@ func (app *App) recoveryChecker(ctx context.Context) {
 		case <-ticker.C:
 			app.checkRecovery()
 			app.checkCrashRecovery()
+			err := app.SetResetupStatus(app.cluster.Local().Host(), app.doesResetupFileExist())
```

Comment:
Let's create two methods - one setResetupStatus() in app.go
And SetResetupStatus(host, exists) in app_dcs.go

And do error handling and args in setResetupStatus()